.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
a ".vscode" entry should be added to the .gitignore to avoid pushing local workspace specifics 😄